ui/src/main/webapp/picsureui/settings/settings.json

```diff
@@ -1,20 +1,26 @@
-
 {
-  "resources":[
-		{
-			"id" : "UDN",
-			"name" : "UDN",
-			"basePath" : "/picsure",
-			"findPath" : "/PIC-SURE/search"
-		}
-		],
-  "helpLink": "about:blank",
-  "advancedSearchLink": "",
-  "picSureResourceId":"02e23f52-f354-4e8b-992c-d37c8b9ba140",
-  "applicationIdForBaseQuery":"8b5722c9-62fd-48d6-b0bf-4f67e53efb2b",
-  "useAsyncQuery": true,
-  "categoryAliases": {},
-  "categorySearchResultList" : [],
-  "logoPath": "/logo.png",
+  "resources": [
```

Comment:
Do we need this 'resources' element?  and if so, it probably shouldn't be called 'datastage'

Reply:
I just copied this from what's working now in development. I'm not 100% familliar with all of these values, but the previous version of this file was definitely not correct. This file generally gets overridden anyway